Dockerfile

```
@@ -1,7 +1,13 @@
-FROM jinaai/jina:master
+FROM jinaai/jina:master as base
 
-ADD *.py *.yml requirements.txt ./
+COPY . ./big_transfer/
+WORKDIR ./big_transfer
 
-RUN pip install -r requirements.txt
+RUN pip install .
 
-ENTRYPOINT ["jina", "pod", "--uses", "config.yml"]
\ No newline at end of file
+FROM base
+RUN pip install -r tests/requirements.txt
+RUN pytest tests
```

```suggestion
RUN pytest -s -v tests
```

for verbose output in CI

---

https://github.com/jina-ai/executor-big-transfer-encoder/pull/1/commits/d1470567f411afdcbb68b5e4294af46cd3d8b527